### DISCUSSION TITLE
SvelteKit

### DISCUSSION THREAD

**Discussion by vhscom** [wpengine/faustjs#1336] [Q&A] [answered]
Any idea when it'll be supported? If it's supported already, anyone created a guide to help others get started?

**Comment by TeresaGobble** [comment, marked as answer]
Hi @vhscom, we don't plan on supporting SvelteKit in the near future. The next several sprints, our focus will be largely on implementation of Gutenberg block support and a few other features. That's not to say we will never support it, but there are no concrete plans or timelines to do so at this time.

**Comment by wilrevehl** [comment]
@TeresaGobble With regard to SvelteKit compatibility (@vhscom mainly only possible with front-end templates as Gutenberg custom blocks require React), it begs the question of whether @faustwp/core truly needs to have the peerDependencies of react and next. Offhand, could you provide some details as to what they're used for? Thanks for your hard work on this!
